src/test/java/org/gridsuite/directory/server/DirectoryTest.java

```diff
@@ -528,7 +528,14 @@ public void testGetElement() {
         ElementAttributes scriptAttributes = toElementAttributes(UUID.randomUUID(), "Script", FILTER, false, "user1");
         insertAndCheckSubElement(rootDirUuid, false, scriptAttributes);
 
-        var res = getElements(List.of(contingencyAttributes.getElementUuid(), filterAttributes.getElementUuid(), scriptAttributes.getElementUuid()), "user1");
+        var res = getElements(List.of(contingencyAttributes.getElementUuid(), filterAttributes.getElementUuid(), UUID.randomUUID()), "user1", true, 404);
+        assertTrue(res.isEmpty());
+
+        res = getElements(List.of(contingencyAttributes.getElementUuid(), filterAttributes.getElementUuid(), UUID.randomUUID()), "user1", false, 200);
```

Comment:
We should expect a HTTP 206 code for a 'Partial Content' return code ?

Reply:
No it doesn't seems to be the 206 code which correspond...

Reply:
207 is for WebDav HTTP extension and need an xml body, it doesn't fit either.
Ok for 200 return code then, but we always need tp check if the result of this Get method is complete which is a pain.